### PR TITLE
Adding Stanford collection parsing.

### PR DESCRIPTION
**Pull request by yoagauthier**

### CODE REVIEW

File: models/document.py [hunk 1]
```diff
@@ -83,6 +92,21 @@ def load_collection(self):
                 )
 
 
+class StanfordDocumentCollection(DocumentCollection):
+    def load_collection(self):
+        for directory in os.listdir(self.data_filename):
+            print("DIRECTORY", directory)
```
Comment (SeysT) [inline models/document.py:98]: You keep this print, is it normal?

File: cacm_questions.py [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
+            load_on_creation=False,
+        )
+        document_collection.load_from_file('Data/Collection/cacm.collection')
+    else:
+        print("TWO")
+        document_collection = CACMDocumentCollection(
+            data_filename='Data/CACM/cacm.all',
+            stop_list_filename='Data/CACM/common_words',
+            load_on_creation=True,
+        )
```
Comment (SeysT) [inline cacm_questions.py:26]: You keep print here but you don't keep it in stanford_question, is it normal?

Comment (yoagauthier) [reply cacm_questions.py:26]: No it's not, I forgot to remove it !

File: models/document.py (hunk 1 above)
Comment (yoagauthier) [reply models/document.py:98]: As the parsing of the collection can be quite long (up to 10 mins), I find it useful and not too verbose to be sure the parsing goes on. Do you think we should remove it ?

Comment (SeysT) [reply models/document.py:98]: No I think it is a good solution, I just wanted to be sure we know why we keep a print in the model :)

**Pull request merged**